fe/fe-core/src/main/java/org/apache/doris/analysis/CreateTableAsSelectStmt.java

```diff
@@ -82,6 +84,10 @@ public void analyze(Analyzer analyzer) throws UserException {
                 queryStmt.getResultExprs().get(i).getSrcSlotRef().getColumn()
                         .setIsAllowNull(outputs.get(i).isNullable());
             }
+            if (Config.disable_datev1 && queryStmt.getResultExprs().get(i).getType() == Type.DATE) {
```

Comment:
i think we should use `Config.enable_date_conversion`?

Reply:
done